### ISSUE TITLE
plot_unit_waveforms places start of each wavform at zero.

### ISSUE THREAD

**Issue by McD-NMI** [SpikeInterface/spikewidgets#48]
I noticed in the plots of my waveforms that the start of each cutout is placed as [0,0] (see image). I am not sure if this makes sense, as this part of the cutout is just noise. This leads to the cutouts looking like they don't overlap, but I expect they would overlap much better if they were all overlaid in another way. For example using the average of the signal as the baseline and placing this on x-axis. I am not sure on the correct way cutouts should be overlaid for electrophysiology, perhaps by the slope of the leading edge makes more sense for example? 

I hope this makes sense! 

![waveform_channel_49](https://user-images.githubusercontent.com/47813652/69853088-386ff380-1286-11ea-9187-2533fd66c65f.png)

**Comment by samuelgarcia** [comment]
A good rule for this is when the MAD come back to 1 and the median come back to 0.

See
https://tridesclous.readthedocs.io/en/latest/important_details.html#waveforms-extraction

**Comment by alejoe91** [comment]
@McD-NMI I don't think that is the case. The cut outs  are extracted based on the spike trains. If they don't look aligned, then the spike sorter did not return aligned waveforms! The widget just use the `spiketoolkit get_unit_waveforms` to extract the waveforms and then it plots them.
